alf/metrics/metrics.py

```diff
@@ -266,6 +283,114 @@ def _extract_metric_values(self, time_step):
             return [reward[..., i] for i in range(reward.shape[-1])]
 
 
+@gin.configurable
+class AverageDiscountedReturnMetric(AverageEpisodicSumMetric):
+    """Metric for computing the average discounted episodic return.
+    It is calculated according to the following formula:
+
+    .. math::
+        \begin{array}{ll}
+            R &= r_1 + (1 + gamma) r_2 + (1 + gamma + gamma^2) r_3 + \cdots \\
```

Comment:
gamma needs backslash; just seen from this first line, it's not the discounted return; either there is some explanation missing or the code is computing something else

Reply:
Fixed.